tests/test_box.py

```diff
@@ -46,12 +47,14 @@ def test_length(box):
 #    assert(bx.num_pts == 129 * 128 * 128)
 #    assert(bx.volume == 128**3)
 
-'''
+
 @pytest.mark.parametrize("dir", [-1, 0, 1, 2])
 def test_surrounding_nodes(box, dir):
     """Surrounding nodes"""
-    nx = np.array(box.hi_vect)
+    nx = box.hi_vect
+    print('nx=', nx)
     bx = box.surrounding_nodes(dir=dir)
+    print('bx=', bx)
 
     if dir < 0:
```

Comment:
Issues below this line for the `test_surrounding_nodes` test

Reply:
Looks like there was some extra logic added for this test:
https://github.com/sayerhs/pyamrex/blob/f2909f90e0c43e45829a7275a77ef009fd726c40/amrex/amrex_base.pyx#L106-L128

Reply:
@ax3l cython doesn't have a clean way to support overloaded functions. So `-1` was used as a proxy for the surrounding nodes method without any arguments. You can probably cleanup the logic in pybind11

Reply:
Oh, thanks a lot for the hint - that's the info I needed :)